public/controllers/agent/components/register-agent.js

```diff
@@ -847,8 +836,90 @@ export const RegisterAgent = withErrorBoundary(
               title='This section could not be displayed because you do not have permission to get access to the registration service.'
               iconType="iInCircle"
             />
-          ) :
-            this.state.selectedOS && (
+          ) : (this.state.connectionSecure === true && this.state.udpProtocol === false) ? (
+            <EuiText>
+            <p>
+              You can use this command to install and enroll the Wazuh agent in one or more hosts.
+            </p>
+            <EuiCallOut
+              color="warning"
+              title={
+                <>
+                  If the installer finds another Wazuh agent in the system, it will upgrade it preserving the configuration.
```

Comment:
As these texts are used several times, I'd extract them as constants, so we improve reusability and makes it easier to maintain over time. Imagine we need to change these texts in the future, instead of changing every occurrence, we'd only need to change it once.

Reply:
Fixed in this commit: [2a582f7](https://github.com/wazuh/wazuh-kibana-app/pull/4776/commits/2a582f7b44d5c90d352c1388ed592969779e69da)